### PR TITLE
Fixed wording domains in multistore dropdown template

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fixes wrong domains for wordings in multishop dropdown's twig template.
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | none
| How to test?      | only wording check is needed, no QA, CI must be green
| Possible impacts? | none

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24028)

### CODE REVIEW

**Comment by Julievrz** [comment]
I think you miss 'Group' in Admin.Global too. :)

File: src/PrestaShopBundle/Resources/views/Admin/Multistore/dropdown.html.twig [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <div class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenuButton4">
       <div class="multistore-dropdown-search-container">
         <i class="material-icons">search</i>
-        <input type="text" class="form-control multistore-dropdown-search js-multistore-dropdown-search" placeholder="Search shop name" data-no-results="{{ "No results found for"|trans({}, 'Admin.Shop') }}" data-searching="{{ "Searching for"|trans({}, 'Admin.Shop') }}">
+        <input type="text" class="form-control multistore-dropdown-search js-multistore-dropdown-search" placeholder="Search shop name" data-no-results="{{ "No results found for"|trans({}, 'Admin.Global') }}" data-searching="{{ "Searching for"|trans({}, 'Admin.Global') }}">
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Multistore/dropdown.html.twig]:
```suggestion
        <input type="text" class="form-control multistore-dropdown-search js-multistore-dropdown-search" placeholder="Search shop name" data-no-results="{{ 'No results found for'|trans({}, 'Admin.Global') }}" data-searching="{{ 'Searching for'|trans({}, 'Admin.Global') }}">
```

File: src/PrestaShopBundle/Resources/views/Admin/Multistore/dropdown.html.twig [hunk 2]
```diff
@@ -22,10 +22,10 @@
                   {% if shop.isConfigurationKeyOverridden(shopConfiguration, configurationKey) == true %}
                     <p class="multistore-dropdown-shop-status multistore-dropdown-shop-status-locked">
                       <i class="material-icons">https</i>
-                      {{ "Customized"|trans({}, 'Admin.Shop') }}
+                      {{ "Customized"|trans({}, 'Admin.Global') }}
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Multistore/dropdown.html.twig]:
```suggestion
                      {{ 'Customized'|trans({}, 'Admin.Global') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Multistore/dropdown.html.twig [hunk 3]
```diff
@@ -22,10 +22,10 @@
...
                   {% else %}
                     <p class="multistore-dropdown-shop-status">
-                      {{ "Inherited"|trans({}, 'Admin.Shop') }}
+                      {{ "Inherited"|trans({}, 'Admin.Global') }}
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Multistore/dropdown.html.twig]:
```suggestion
                      {{ 'Inherited'|trans({}, 'Admin.Global') }}
```

**Review by Julievrz — COMMENTED** [review]
I'm not sure it changes anything but, just in case. :)

**Comment by matthieu-rolland** [comment]
> I think you miss 'Group' in Admin.Global too. :)

I think it's good? It was already `Admin.Global`   (all the others were wrong though)

I applied your quote modifications, it doesn't change the behavior, but it is cleaner 👍

**Comment by Julievrz** [comment]
> > I think you miss 'Group' in Admin.Global too. :)
> 
> I think it's good? It was already `Admin.Global` (all the others were wrong though)
> 
> I applied your quote modifications, it doesn't change the behavior, but it is cleaner 👍

That's good, thank you @matthieu-rolland!

**Review by NeOMakinG — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi @matthieu-rolland !

I checked the wording, it's OK !

pr24028 ok

QA :heavy_check_mark:

**Comment by Progi1984** [comment]
Thanks @matthieu-rolland & @Robin-Fischer-PS

**Pull request merged**